implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/AggregationTemporality.java

```diff
@@ -48,4 +48,12 @@ public static io.opentelemetry.proto.metrics.v1.AggregationTemporality toOtlpAgg
         }
     }
 
+    static boolean isDelta(io.opentelemetry.proto.metrics.v1.AggregationTemporality aggregationTemporality) {
```

Comment:
I don't think we should make these decisions directly based on `io.opentelemetry.proto.metrics.v1.AggregationTemporality` and always based on `io.micrometer.registry.otlp.AggregationTemporality` otherwise what is the point of this enum?

I remember I made this change in your previous PR as well, could you please revert it back so that we always depend on the Micrometer enums and not the OTel fields?